src/daffodilDebugger/utils.ts

```diff
@@ -78,9 +90,11 @@ export async function runDebugger(
     scriptPath,
     artifact.scriptName,
     createTerminal,
-    shellArgs(artifact.scriptName, serverPort),
+    shellArgs(serverPort),
     {
       DAFFODIL_DEBUG_CLASSPATH: daffodilDebugClasspath,
+      DAFFODIL_DEBUGGER_LOG_LEVEL: debuggerLogLevel,
```

Comment:
Oops my suggestion doesn't match the classpath one, sorry... adjust to match?

Reply:
Yeah I will do that with some of the other updates